packages/lit-dev-content/site/tutorials/content/custom-attribute-converter/00.md

```diff
@@ -0,0 +1,33 @@
+Web Components can receive data in many ways, but one of the most commmon ways is to use an attribute because attributes are declarative, static, and easy to read & write in HTML!
+
+Every HTML Element in the browser has *attibutes* and *properties*. [*Attributes*](https://developer.mozilla.org/en-US/docs/Glossary/attribute) are additional values that describe the state of an element in HTML. [*Properties*](https://developer.mozilla.org/en-US/docs/Glossary/property/JavaScript) are the current values of an element's Javascript DOM object.
+
+For example, the `<input>` element has both a `value` attribute and a `value` property.
+
+```html
+<input value="This is setting the 'value' attribute">
+<script>
+  const inputElementReference = document.querySelector('input');
+  inputElementReference.value = 'This is setting the "value" property';
+</script>
+```
+
+Since Lit elements are HTML elements, Lit elements have both attributes and properties. Lit can convert given attributes into properties, but some properties will not map easily to an attribute making it difficult to write the attribute. An example would be the `Date` object – you can see an example of this in the playground preview.
+
+Custom attribute converters address this problem.
+
+## What you'll learn
+
+- What is an attribute converter
+- How Lit's built-in attribute converters work
+- How to write your own attribute converter
+
+## What you'll build
+
+In this tutorial you will build a component that takes in a human-readable date-string, converts to a Javascript Date object, and displays the given date in a human readable manner.
+
+You will do this with the default attribute converters and then upgrade your component with your own custom attribute converter that can convert a date-string attribute into a Javascript Date object property and vice versa.
+
+## Previous knowledge
+
+It's recommended you understand the basics of [reactive properties](/docs/components/properties/) in Lit.
```

Comment:
Should [lifecycle](https://lit.dev/docs/components/lifecycle/) be considered previous knowledge as well considering the use of `willUpdate`? I do see that it is linked in the further reading section.